### ISSUE TITLE
Brew install doesn't work for xcode 13?

### ISSUE THREAD

**Issue by ronnyli** [babashka/babashka#1020]
**version**

N/A - not installed

**platform**

Mac OSX 11.6 Big Sur

**problem**
Maybe this is a known issue but I tried installing via brew on Xcode version 13, which seemed to result in an error:

<img width="449" alt="Screen Shot 2021-09-27 at 3 49 52 PM" src="https://user-images.githubusercontent.com/6841011/134975533-b6016ce5-2722-4ba3-b83b-54f23ec0cbd2.png">

**repro**

See above

**expected behavior**

expected `brew install` to succeed

**Comment by borkdude** [comment]
I think this is a brew problem. babashka doesn't need xcode at all.

**Comment by borkdude** [comment]
Btw, until you sort this out, you can also just use the installer script in this repo (see README).

**Comment by ronnyli** [comment]
Thanks! That's what I ended up doing :)

**Comment by borkdude** [comment]
I don't think this is a bb issue that we can solve on our end, so I'm closing this until further notice.
